lib/workers/repository/update/pr/index.ts

```diff
@@ -82,16 +89,34 @@ export async function ensurePr(
   );
 
   const config: BranchConfig = { ...prConfig };
-
+  const prFingerprint = fingerprint(config);
   logger.trace({ config }, 'ensurePr');
   // If there is a group, it will use the config of the first upgrade in the array
   const { branchName, ignoreTests, prTitle = '', upgrades } = config;
   const dependencyDashboardCheck =
     config.dependencyDashboardChecks?.[config.branchName];
-  // Check if existing PR exists
+  // Check if PR already exists
   const existingPr = await platform.getBranchPr(branchName);
   if (existingPr) {
     logger.debug('Found existing PR');
+    const prCache = getPrCache(branchName);
+    if (prCache) {
+      logger.debug({ prCache }, 'Found existing Pr cache');
+      const lastEditTime = prCache.lastEdited;
+      // return if pr cache is valid and pr was last edited before a day 
+      if (
+        isCloned() === false &&
+        prFingerprint === prCache.fingerprint &&
+        getElapsedHours(lastEditTime) > 24
+      ) {
+        logger.debug('Pr fingerprints match, skiping fetching changelogs');
+        return { type: 'with-pr', pr: existingPr };
+      }
+      logger.debug('Pr fingerprints do not match');
```

Comment:
This will be misleading at times, because fingerprint could match and you still reach here

Reply:
Maybe just logging that the cache can't be used ie. invalid is a better choice?
```ts suggestion
{
        logger.debug('Cache is valid, skipping PR update');
        return { type: 'with-pr', pr: existingPr };
      }
      logger.debug('Invaild cache, processing PR');
```